anchor/programs/pubkey-protocol/src/instructions/identity/remove.rs

```diff
@@ -22,7 +22,7 @@ pub struct RemoveIdentity<'info> {
 
     #[account(
       mut,
-      seeds = [&Pointer::hash_seed(&pointer.provider, &args.provider_id)],
+      seeds = [&Pointer::hash_seed(&args.provider, &args.provider_id)],
```

Comment:
I think we don't have to pass this in through `args` because we supply the `pointer` and take it from there.